pywbem/_cim_obj.py

```diff
@@ -368,7 +368,7 @@ def _qualifiers_tomof(qualifiers, indent, maxline=MAX_MOF_LINE):
     line_pos = indent + 1
 
     mof_quals = []
-    for q in qualifiers.itervalues():
+    for q in qualifiers.values():
```

Comment:
Was there a reason for this one to move to values rather than six.itervalues??

Reply:
My thinking was to use the values() method that is recommended for python 3. However, in python 2 that results in a small loss of efficiency compared to using itervalues(). The one advantage of the dictionary view returned by values() in python 3 (it can iterate multiple times) is not needed here, so I think going to six.itervalues() is actually a good recommendation.

Reply:
TODO(Andy): Add comment explaining why we use values() in this case.

Reply:
DONE - I just clarified the content of the qualifiers dict, that should be sufficient.